.github/workflows/ci.yml

```diff
@@ -46,24 +47,42 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: Modify maven settings
+        if: ${{ env.USE_STAGE == 'true' }}
+        run: |
+          sed -i 's/<\/settings>/<profiles><profile><id>local-repo<\/id><repositories><repository><id>central<\/id><url>https:\/\/repo.maven.apache.org\/maven2<\/url><releases><enabled>true<\/enabled><\/releases><snapshots><enabled>false<\/enabled><\/snapshots><\/repository><repository><id>staged-releases<\/id><url>https:\/\/repository.apache.org\/content\/groups\/staging\/<\/url><\/repository><\/repositories><pluginRepositories><pluginRepository><id>staged-releases<\/id><url>https:\/\/repository.apache.org\/content\/groups\/staging\/<\/url><\/pluginRepository><\/pluginRepositories><\/profile><\/profiles><activeProfiles><activeProfile>local-repo<\/activeProfile><\/activeProfiles><\/settings>/' $HOME/.m2/settings.xml
+
+      - name: copy maven settings
+        run: |
+          cp $HOME/.m2/settings.xml /tmp/settings.xml
```

Comment:
shall we copy it in the 1st step？（seems one time modification for it is enough）

and we just specify the path in  java maven action